modules/openapi-generator/src/test/resources/3_0/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml

```diff
@@ -1855,6 +1855,7 @@ components:
           - $ref: '#/components/schemas/banana'
     fruitReq:
        oneOf:
+          - type: 'null'
```

Comment:
Can you add this to the v3 python-experimental spec also?

Reply:
done